src/components/objectListing/table/table.component.tsx

```diff
@@ -215,27 +267,18 @@ export const Table = ({
           </tr>
         )}
         {virtualRows.map((virtualRow) => {
-          const row = rows[virtualRow.index] as Row<ParsedSkylarkObject>;
-          const { uid, objectType } = row.original;
           return (
-            <tr
-              key={row.id}
-              className="group/row h-6 align-middle outline-none md:h-10"
-              tabIndex={-1}
-              onDoubleClick={() => setPanelObject?.({ uid, objectType })}
-            >
-              {row.getVisibleCells().map((cell) => (
-                <TableData
-                  tableMeta={tableMeta}
-                  key={cell.id}
-                  cell={cell}
-                  withCheckbox={withCheckbox}
-                />
-              ))}
-            </tr>
+            <TableRow
+              key={virtualRow.index}
+              rows={rows}
```

Comment:
can we just pass in the `row` instead of all the `rows`

Reply:
of course... 🤦 good point